recipes/pillow-jpls/meta.yaml

```diff
@@ -0,0 +1,74 @@
+{% set name = "pillow-jpls" %}
+{% set version = "1.2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pillow-jpls-{{ version }}.tar.gz
+  sha256: 4b1b285ecc59d1a31110cfb4077e31e4e291b4694fca5a226afbf460be9870ac
+  patches:
+    - remove_conan.diff
+    # - update_lib_names.diff
+
+build:
+  number: 0
+  skip: true  # [osx]
+
+requirements:
+  build:
+    - python                                 # [build_platform != target_platform]
+    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - make  # [unix]
+    - ninja  # [win]
+    - cmake
+    - scikit-build
+    - pybind11
+    - pybind11-abi
+    - charls
+    - fmt
+    - eigen
+  host:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - pip
+    - python
+    - scikit-build
+    - pybind11
+    - pybind11-abi
+    - charls
+    - fmt
+    - eigen
```

Comment:
The build section is for build tools like compilers and cmake. The host section is for libraries and tools that must match the target platform like libraries and python/pip.